NuGet.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+  </packageSources>
+  <config>
+    <add key="repositoryPath" value=".nuget\packages" />
+    <add key="globalPackagesFolder" value=".nuget\packages" />
```

Comment:
One is for packages.config and the other is for project.json and package ref.  I keep both so I don't have to remember/think about which is which and I can copy the same pattern to places where different nuget paradigms are used. I don't know why nuget has this distinction. I also don't know why we need 4 environment variables with this value yet again in build.ps1 and build.sh, but this is in keeping with that. ;)